h_periodic/h_beat.py

```diff
@@ -1,9 +1,25 @@
 """Celery beat scheduler process configuration."""
 
+import sys
 from datetime import timedelta
+from os import environ
 
 from celery import Celery
 
+
+def asbool(s):
+    """Return True if s is any of "t", "true", "y", etc (case-insensitive)."""
+    return str(s).strip().lower() in ("t", "true", "y", "yes", "on", "1")
```

Comment:
I get where this is coming from, but it feels like overkill. I think we'd be better off just documenting that it works for any truthy string including "No" or "0". We're not creating a framework here and I this feels a bit disproportionate.

It also invites the whole why doesn't `DISABLE_H_BEAT=disable` (or any other hair brained scheme) work etc.

Reply:
These are the same string values as Pyramid's `asbool` accepts as truthy, so `DISABLE_H_BEAT` works the same as other boolean envvars in our other apps. It's probably a good idea to make it accept the same strings as the envvars in our other apps do. For example changing it to `"false"` seems like a likely way that someone would try to turn it off, and that would actually turn it on if we just used `bool(s)`. It's one line of code so I think we can get away with it

Reply:
> It's only one line of code...

Hmm.... every line of code is only one line of code if you don't count the others. Tests are code.

A `strip()` would get you the rest.

Reply:
Let's say I changed it to:

    return bool(str(s).strip())

You would have to put a value into the envvar to make it truthy (since any empty string would be falsey), so you'd probably do the same as we do for our other boolean envvars:

    DISABLE_H_BEAT=true

Now if someone comes along and wants to disable it without deleting the envvar they could do so by changing it to an empty string but when they see `true` I think they're very likely to change it to:

    DISABLE_H_BEAT=false

That's intuitive and is also what we do for our other boolean envvars. But with `bool(str(s).strip())` that would be truthy.

I think I'd like to stick with `str(s).strip().lower() in ("t", "true", "y", "yes", "on", "1")`, it's only 33 chars-of-code longer and gives this envvar the same behaviour as the ones in our other apps. It doesn't _really_ require any more test code either, just more cases in the parametrize, and they're very very simple tests.